### PR TITLE
Launch celery directly rather than via django-celery

### PR DESCRIPTION
**Pull request by timmc-edx**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/roles/notifier/templates/edx/app/supervisor/conf.d/notifier-celery-workers.conf.j2 [hunk 1]
```diff
@@ -9,7 +9,7 @@ environment={% for name,value in notifier_env_vars.items() -%}
             {%- endfor -%}
             PID="/var/tmp/notifier-scheduler.pid",LANG="en_US.UTF-8",DJANGO_SETTINGS_MODULE="notifier.settings_local"
 
-command={{ NOTIFIER_VENV_DIR }}/bin/python manage.py celery worker -l {{ NOTIFIER_LOG_LEVEL }} --hostname=notifier.%%h
+command={{ NOTIFIER_VENV_DIR }}/bin/celery worker -l {{ NOTIFIER_LOG_LEVEL }} --hostname=notifier.%%h
```
Comment (jmbowman) [inline playbooks/roles/notifier/templates/edx/app/supervisor/conf.d/notifier-celery-workers.conf.j2]: This is just for the celery workers of the notifier service, which I think is about to be decommissioned (@bderusha ?).  It looks like the workers in edx-platform are started via https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/templates/edx/app/edxapp/worker.sh.j2 , which in turn is called by https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/templates/edx/app/supervisor/conf.d.available/workers.conf.j2 .

Comment (timmc-edx) [reply playbooks/roles/notifier/templates/edx/app/supervisor/conf.d/notifier-celery-workers.conf.j2]: Ah! So it is. Thanks for the pointers.

**Review by feanil — APPROVED** [review]

**Comment by timmc-edx** [comment]
From discussion with @jmbowman: This might not work as intended, and I should probably be setting `--app={{ w.service_variant }}.celery` instead.

I'll build a sandbox from the branch to confirm. If changes are needed here, they'll be needed for https://github.com/edx/edx-platform/pull/23222 as well.

File: playbooks/roles/edxapp/templates/edx/app/supervisor/conf.d.available/workers.conf.j2 [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
-environment={% if COMMON_ENABLE_NEWRELIC_APP %}NEW_RELIC_APP_NAME={{ EDXAPP_NEWRELIC_WORKERS_APPNAME }},NEW_RELIC_DISTRIBUTED_TRACING_ENABLED={{ EDXAPP_WORKERS_ENABLE_NEWRELIC_DISTRIBUTED_TRACING }},NEW_RELIC_LICENSE_KEY={{ NEWRELIC_LICENSE_KEY }},{% endif -%}CONCURRENCY={{ w.concurrency }},LOGLEVEL=info,DJANGO_SETTINGS_MODULE={{ worker_django_settings_module }},LANG={{ EDXAPP_LANG }},PYTHONPATH={{ edxapp_code_dir }},SERVICE_VARIANT={{ w.service_variant }},BOTO_CONFIG="{{ edxapp_app_dir }}/.boto,EDX_REST_API_CLIENT_NAME=edx.{{ w.service_variant }}.core.{{ w.queue }},"
+environment={% if COMMON_ENABLE_NEWRELIC_APP %}NEW_RELIC_APP_NAME={{ EDXAPP_NEWRELIC_WORKERS_APPNAME }},NEW_RELIC_DISTRIBUTED_TRACING_ENABLED={{ EDXAPP_WORKERS_ENABLE_NEWRELIC_DISTRIBUTED_TRACING }},NEW_RELIC_LICENSE_KEY={{ NEWRELIC_LICENSE_KEY }},{% endif -%}CONCURRENCY={{ w.concurrency }},LOGLEVEL=info,DJANGO_SETTINGS_MODULE={{ w.service_variant }}.envs.{{ worker_django_settings_module }},LANG={{ EDXAPP_LANG }},PYTHONPATH={{ edxapp_code_dir }},SERVICE_VARIANT={{ w.service_variant }},BOTO_CONFIG="{{ edxapp_app_dir }}/.boto,EDX_REST_API_CLIENT_NAME=edx.{{ w.service_variant }}.core.{{ w.queue }},"
```
Comment (timmc-edx) [inline playbooks/roles/edxapp/templates/edx/app/supervisor/conf.d.available/workers.conf.j2:4]: Changed to use full module path to `DJANGO_SETTINGS_MODULE`.

**Review by jmbowman — APPROVED** [review]

**Pull request merged**